tutor/src/screens/gradebook/export-btn.js

```diff
@@ -36,14 +44,27 @@ class ScoresExport extends React.Component {
 
   render() {
     return (
-      <TourAnchor className="job scores-export" id="scores-export-button">
-        <Icon
-          type="download"
+      <>
+        <Button
+          ref={this.overlay}
           disabled={this.scoresExport.isPending}
           onClick={this.startExport}
-        />
-        {this.message}
-      </TourAnchor>
+          onMouseEnter={() => 
```

Comment:
I'm not sure if there would be a unwanted tradeoff here, but I usually approach this need with an `OverlayTrigger` wrapper, which can feel simpler than using refs.

Reply:
I tried the `OverlayTrigger`, but the popover was popping out randomly. That was last week, I will check it again.